fvp/spmc_el1_partitions_manifest.dts

```diff
@@ -13,13 +13,19 @@
 	attribute {
 		spmc_id = <0x8000>;
 		maj_ver = <0x1>;
-		min_ver = <0x0>;
+		min_ver = <0x1>;
 		exec_state = <0x0>;
 		load_address = <0x0 0x6000000>;
 		entrypoint = <0x0 0x6000000>;
 		binary_size = <0x80000>;
 	};
 
+	/* Secure world memory */
+	memory@6000000 {
+		device_type = "memory";
+		reg = <0x0 0x6000000 0x1000000>; /* Trusted DRAM */
+	};
```

Comment:
Is this consumed by OP-TEE SPMC?

Reply:
Currently not, but I assume Hafnium does and it would be a good idea to do the same when OP-TEE is the SPMC.

Reply:
Again I can fix that for spmc_el1_partitions_manifest.dts, but I assumed the load address used in that file to load trusted services partitions(driven from fvp-psa-sp.mk) must not overlap. Sorry I don't understand all the details of these manifest files. I assumed either the copy here or in TF-A are up-to-date but it looks like they are not :(